scripts/quests/sandoria/WHM_AF1_Messenger_From_Beyond.lua

```diff
@@ -0,0 +1,100 @@
+-----------------------------------
+-- Messenger from Beyond
+-- !addquest: 0 87
+-- Narcheral: !gotoid 17723585
+-- qm2      : !gotoid 17199727
+
+-----------------------------------
```

Comment:
Small nit, got an empty line and double section break here. You've got the double section breaks in all the quest luas actually.

Reply:
Updated- good catch